src/components/FormDrawer.tsx

```diff
@@ -72,12 +72,13 @@ class BaseFormDrawer extends Component<IProps> {
         visible={isVisible.isTrue}
         width={width}
       >
-        <Form layout='vertical' hideRequiredMark onSubmit={this.formManager.onSave}>
```

Comment:
This is something we should discuss at some points.
I know we moved fast and that's fine, but the friction I often encounter with **fields-ant** is that I can't configure my components like how Ant Design is allowing me. Forcing me to modify fields-ant to change the hardcoded prop or allow me to pass in the Ant Design component's props.

It's an easy fix when we have 1 -> 1 components (`<ButtonXYZ />` for example). But when we have nested components like this, we can't simply allow one set of props. In that specific case we could allow `<FormDrawer>` to have extended `<Antd.Drawer />` props, but then what about the `<Antd.Form />` props ? We should try to find a way to allow it easily.